crates/forge/tests/integration/spy_events.rs

```diff
@@ -428,7 +428,7 @@ fn event_emitted_wrong_data_asserted() {
     assert_case_output_contains!(
         result,
         "test_assert_wrong_data",
-        "2814149079029271315825228546613746536844456515452335425622822705028162471071"
+        "385731233638135702203933925049152883559461865803221708830049339625617856128"
```

Comment:
Why did it change?

Reply:
https://github.com/foundry-rs/starknet-foundry/pull/1121#discussion_r1394072646